### PR TITLE
ACL 2023 misc corrections and workshops

### PR DESCRIPTION
**Pull request by mjpost**
I will accumulate a few corrections here—please let me merge.

Here are all the workshops and their ingestion status:
- [x] BioNLP
- [x] CODI
- [x] RepL4NLP
- [x] WOAH
- [x] DialDoc
- [x] IWSLT
- [x] Natural Language Reasoning and Structured Explanations
- [ ] *SEM (no watermarks)
- [x] SemEval
- [x] SustaiNLP
- [x] LAW
- [x] BEA
- [ ] Workshop on Matching From Unstructured and Structured Data (no watermarks)
- [ ] NLP4ConvAI (not built, no watermarks)
- [ ] TrustNLP (Workshop on Trustworthy NLP) (not built)
- [ ] WASSA (not found)
- [x] ClinicalNLP
- [x] SICon
- [x] CAWL 
- [x] AmericasNLP (The Third Workshop on NLP for Indigenous Languages of the Americas)
- [x] The Fifth Workshop on Narrative Understanding
- [x] SIGMORPHON

### CODE REVIEW

**Comment by mjpost** [comment]
Can someone approve this? @anthology-assist @mbollmann @akoehn @nschneid

File: data/xml/2023.dialdoc.xml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      <url hash="4ba723c4">2023.dialdoc-1.7</url>
+      <bibkey>ma-etal-2023-sldt</bibkey>
+    </paper>
+    <paper id="8">
+      <title>A Dialogue System for Assessing Activities of Daily Living: Improving Consistency with Grounded Knowledge</title>
+      <author><first>Zhecheng</first><last>Sheng</last></author>
+      <author><first>Raymond</first><last>Finzel</last><affiliation>University of Minnesota - Twin Cities</affiliation></author>
+      <author><first>Lucke096@umn.edu</first><last>Lucke096@umn.edu</last><affiliation>NA</affiliation></author>
```
Comment (akoehn) [inline data/xml/2023.dialdoc.xml:109]: There are several authors that are only an email address

**Review by akoehn — APPROVED** [review]
You are in luck, woke up at 5:20 to approve your PRs :-)

We can merge it now but maybe you can change the incorrect names -- I'm on my phone so I cannot look them up from the pdf.

**Comment by mjpost** [comment]
Thanks, Arne!

**Pull request merged**